client/components/web-preview/style.scss

```diff
@@ -34,6 +34,7 @@
 		right: 0;
 		bottom: 0;
 	display: block;
+	background: fade-out( $gray-light, .2 );
```

Comment:
Actually you can do:

`rgba( $gray-light, 0.5)`

and it works. I'm always for using color variables!